NEWS.md

```diff
@@ -1,3 +1,7 @@
+# admiral 0.8.3
+
+- Fixed an issue where CRAN identified a failing test when "a strict Latin-1* locale" is used. (#1469)
+
```

Comment:
This somewhat looks funny as the exact same text is written in the prior version. Either we should only keep the text in the prior version or indicate that the prior attempt actually did not work.

Reply:
switched up slightly